apps/web/lib/team/[slug]/getServerSideProps.tsx

```diff
@@ -175,6 +185,25 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
     } as const;
   }
 
+  const organizationSettings = isOrgContext
+    ? team?.isOrganization
+      ? team?.organizationSettings
+      : team?.parent?.organizationSettings
+    : null;
+  let allowSEOIndexing = false;
+
+  if (!!organizationSettings) {
+    allowSEOIndexing = organizationSettings.allowSEOIndexing;
+
+    if (
+      team?.isOrganization &&
+      organizationSettings.orgProfileRedirectsToVerifiedDomain &&
+      organizationSettings.orgAutoAcceptEmail
+    ) {
+      return getRedirectToVerifiedDomain({ verifiedDomain: organizationSettings.orgAutoAcceptEmail });
+    }
+  }
+
```

Comment:
Can this be moved before, to say line 93? We should prefer early returns, no point doing extra stuff if we are going to be redirecting

Reply:
there is no much execution above this since it branches only in case of `if (!team)`
Anyway organized to have all redirects / returns at the begining at one place.

Reply:
there is no much execution above this and branches out and executes more lines of code only in case of `!team` or no team found, anyway moved above and organized to have all redirects at the beginning